Jenkinsfile

```diff
@@ -6,8 +6,7 @@ buildMvn {
   doDocker = {
     buildJavaDocker {
       publishMaster = 'yes'
-      healthChk = 'yes'
-      healthChkCmd = 'wget --no-verbose --tries=1 --spider http://localhost:8081/admin/health || exit 1'
+      //healthChk for /admin/health in MainVerticleTest.java
```

Comment:
it is ok to disable this check, because on Jenkins there is no Kafka server. Instead we should use separate integration test for health check that is already implemented in MainVerticleTest.java.

The same approach already applied in other modules:
https://github.com/folio-org/mod-data-export-spring/blob/v2.0.0/src/test/java/org/folio/des/InstallUpgradeIT.java
https://github.com/folio-org/mod-inventory-storage/blob/v26.0.0/src/test/java/org/folio/rest/InstallUpgradeIT.java
https://github.com/folio-org/mod-orders/blob/v12.6.0/src/test/java/org/folio/rest/InstallUpgradeIT.java
https://github.com/folio-org/mod-inventory/blob/v20.0.0/src/test/java/it/InventoryIT.java